tutorials/MXNetONNXExport.ipynb

```diff
@@ -0,0 +1,442 @@
+      "\n",
+      "export_model(sym, params, input_shape, input_type=<type 'numpy.float32'>, onnx_file_path=u'model.onnx', verbose=False)\n",
+      "    Exports the MXNet model file, passed as a parameter, into ONNX model.\n",
+      "    Accepts both symbol,parameter objects as well as json and params filepaths as input.\n",
+      "    Operator support and coverage - https://cwiki.apache.org/confluence/display/MXNET/ONNX\n",
+      "    \n",
+      "    Parameters\n",
+      "    ----------\n",
+      "    sym : str or symbol object\n",
+      "        Path to the json file or Symbol object\n",
+      "    params : str or symbol object\n",
+      "        Path to the params file or params dictionary. (Including both arg_params and aux_params)\n",
+      "    input_shape : List of tuple\n",
+      "        Input shape of the model e.g [(1,3,224,224)]\n",
+      "    input_type : data type\n",
+      "        Input data type e.g. np.float32\n",
+      "    onnx_file_path : str\n",
+      "        Path where to save the generated onnx file\n",
+      "    verbose : Boolean\n",
+      "        If true will print logs of the model conversion\n",
+      "    \n",
+      "    Returns\n",
+      "    -------\n",
+      "    onnx_file_path : str\n",
+      "        Onnx file path\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "help(onnx_mxnet.export_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the API description, you can see that `export_model` API accepts 2 kinds of inputs:\n",
+    "\n",
+    "### MXNet sym, params objects:\n",
+    "\n",
+    "    This is useful if you are training a model. At the end of training, you just need to invoke the `export_model` function and provide sym and params objects as inputs with other attributes to save the model in ONNX format.\n",
+    "    \n",
+    "### MXNet's exported json and params files:\n",
+    "\n",
+    "    This is useful if you have pretrained models and you want to convert them to ONNX format.\n",
+    "\n",
+    "In this tutorial, we will show second usecase: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Downloaded input symbol and params files\n",
+    "sym = 'resnet-18-symbol.json'\n",
+    "params = 'resnet-18-0000.params'\n",
+    "# Standard Imagenet input - 3 channels, 224*224\n",
+    "input_shape = (1,3,224,224)\n",
+    "# Path of the output file\n",
+    "onnx_file = 'mxnet_exported_resnet50.onnx'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Invoke export model API. It returns path of the converted onnx model\n",
+    "converted_model_path = onnx_mxnet.export_model(sym, params, [input_shape], np.float32, onnx_file)"
```

Comment:
trying it out on mxnet 1.3.0 (latest), getting an error:
```
ValidationError: Input index 3 must be set to consumed for operator BatchNormalization

==> Context: Bad node spec: input: "data" input: "bn_data_gamma" input: "bn_data_beta" input: "bn_data_moving_mean" input: "bn_data_moving_var" output: "bn_data" name: "bn_data" op_type: "BatchNormalization" attribute { name: "epsilon" f: 2e-05 type: FLOAT } attribute { name: "momentum" f: 0.9 type: FLOAT } attribute { name: "spatial" i: 0 type: INT }
```

Reply:
Was an issue on my local env, had an older version of ONNX.
Tested with ONNX 1.2.2 and it works as expected.